packages/shared/src/image/NetworkImage.js

```diff
@@ -41,11 +40,14 @@ export const NetworkImage = function NetworkImage(imageProps: Props) {
       },
     ],
   };
+
   return <Image {...newProps} />;
 };
 
+const DefaultIndicator = () => null;
+
 NetworkImage.defaultProps = {
-  indicator: IconLoading,
+  indicator: DefaultIndicator,
```

Comment:
So no visual indication that something is loading, right?

Reply:
Yes, the designers exact works:

`I would remove the loading indicator on the hotel images (they are quite distracting, I would use just plain background with *Cloud light* color)`